app/utils/caching.js

```diff
@@ -0,0 +1,39 @@
+
+// # caching
+var _ = require('underscore');
+var _str = require('underscore.string');
+var path = require('path');
+var helmet = require('helmet');
+
+_.mixin(_str.exports());
+
+exports = module.exports = function(IoC) {
+  return {
+    getCachingHeadersFromSettings: function(settings){
+      var maxAge = parseInt(settings.staticServer.maxAge);
```

Comment:
Please add the `parseInt()`'s second argument. `var maxAge = parseInt(settings.staticServer.maxAge, 10);`